libs/designer-ui/src/lib/html/index.tsx

```diff
@@ -24,6 +24,7 @@ export const HTMLEditor = ({ placeholder, readonly, initialValue, getTokenPicker
       initialValue={initialValue}
       getTokenPicker={getTokenPicker}
       onBlur={handleBlur}
+      label={label}
```

Comment:
This prop should not affect UI; please see reply to first comment

Reply:
This is a first class prop in any editor component, we have not defined it because we decided to not put label ui inside editor. Once you put this the editor interface suggest to use it in UI and it can be mistakently used in future.

provide a first class prop for your automation id or for the label, like we would use ariaLabel for just arialabel and not reuse label if not needed. I think the first design was ok compared to the confusion this can cause.

Reply:
I understand. Thank you for sharing. I will remove the `label`s being passed in everywhere